docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -0,0 +1,331 @@
+W0210 15:02:09.904082    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+W0210 15:02:09.906932    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+W0210 15:02:09.929946    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+W0210 15:02:09.930379    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+NAME: tempo
+LAST DEPLOYED: Fri Feb 10 15:02:08 2023
+NAMESPACE: tempo-test
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+***********************************************************************
+ Welcome to Grafana Tempo
+ Chart version: 1.0.1
+ Tempo version: 2.0.0
+***********************************************************************
+
+Installed components:
+* ingester
+* distributor
+* querier
+* query-frontend
+* compactor
+* memcached
+```
+
+>**NOTE**: If you update your `values.yaml` or `custom.yaml`, run the same helm install command and replace `install` with `upgrade`.
+
+Check the statuses of the Tempo pods:
+
+```bash
+kubectl -n tempo-test get pods
+```
+
+The results look similar to this:
+
+```
+NAME                                    READY   STATUS    RESTARTS   AGE
+tempo-compactor-86cd974cf-8qrk2         1/1     Running   0          22h
+tempo-distributor-bbf4889db-v8l8r       1/1     Running   0          22h
+tempo-gateway-75f468bd46-fbzj4          1/1     Running   0          22h
+tempo-ingester-0                        1/1     Running   0          22h
+tempo-ingester-1                        1/1     Running   0          22h
+tempo-ingester-2                        1/1     Running   0          22h
+tempo-memcached-0                       1/1     Running   0          8d
+tempo-minio-6c4b66cb77-sgm8z            1/1     Running   0          26h
+tempo-querier-777c8dcf54-fqz45          1/1     Running   0          22h
+tempo-query-frontend-7f7f686d55-xsnq5   1/1     Running   0          22h
+```
+
+Wait until all of the pods have a status of Running or Completed, which might take a few minutes.
+
+## Next step
+The next step is to test your Tempo installation by sending trace data to Grafana. You can use the [Set up a test application for a Tempo cluster](https://grafana.com/docs/tempo/latest/setup/set-up-test-app/) document for step-by-step instructions.
+
+If you already have Grafana available, you can add a Tempo data source using the URL fitting to your environment. For example:
+`http://tempo-query-frontend.trace-test.svc.cluster.znet:3100`
```

Comment:
change `cluster.znet` to `cluster.local` :)

Reply:
Done. 
```suggestion
`http://tempo-query-frontend.trace-test.svc.cluster.local:3100`
```